lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
> No, one half of the contacts are deleted via delete_contacts_of_me and the second half is deleted with deleting aspects in delete_standard_user_associations

No, `delete_contacts_of_me` are contacts of local people who have us in their aspects. The thing that isn't deleted are contacts where `user` is us, and `person` is someone else, while `delete_contacts_of_me` deletes contacts where we're a person. So that's a different thing.